tools/SampleAnalyzer/Interfaces/delphes/DelphesTreeReader.cpp

```diff
@@ -562,7 +562,9 @@ void DelphesTreeReader::FillEvent(EventFormat& myEvent, SampleFormat& mySample)
   if (data_.Jet_!=0)
   {
     MAuint32 njets = static_cast<MAuint32>(data_.Jet_->GetEntries());
-    myEvent.rec()->jets_.reserve(njets);
+//    myEvent.rec()->jets_.reserve(njets);
+      std::vector<RecJetFormat> output_jets;
```

Comment:
@BFuks This portion is the problematic part with delphes. Since we don't have `jet_` anymore, everything needs to go in `jetcollection_` however, I'm getting segfault during delphes test, and I don't understand why. Can you check it out?

PS: After this problem is solved, I'll also mitigate `fatjet` and `genjet` into `jetcollection_`. The accessors will act the same way.

Reply:
Can you please let me know whether the problem is solved or not? I have lost track a bit. If it is not, please send me a simple example in which it crashes and I will deal into it. Remember that for now, I haven't checked the shared library branch. Therefore, if it is solved in the latter, please let me know and ignore this reply to your comment from 3 weeks ago.

Reply:
> Can you please let me know whether the problem is solved or not? I have lost track a bit. If it is not, please send me a simple example in which it crashes and I will deal into it. Remember that for now, I haven't checked the shared library branch. Therefore, if it is solved in the latter, please let me know and ignore this reply to your comment from 3 weeks ago.

This has been resolved in the `sharedlib` branch. You can test it as well with delphes on your side it was just a compilation issue.